### PR TITLE
LLM Module Readme Update

### PR DESCRIPTION
**Pull request by anoopshrma**
# Description

Updated Readme for LLM modules **✅**

Done with LLM module, will pick another module in sometime 🤓

### CODE REVIEW

**Comment by anoopshrma** [comment]
Hey @logan-markewich 👋  got a big PR for you 😅

File: llama-index-integrations/llms/llama-index-llms-anthropic/pyproject.toml [hunk 1]
```diff
@@ -27,7 +27,7 @@ exclude = ["**/BUILD"]
 license = "MIT"
 name = "llama-index-llms-anthropic"
 readme = "README.md"
-version = "0.3.4"
+version = "0.3.5"
```
Comment (logan-markewich) [inline llama-index-integrations/llms/llama-index-llms-anthropic/pyproject.toml:30]: We actually don't need to vbump these for README updates -- is it easy to undo these? 😅

**Review by logan-markewich — COMMENTED** [review]
I skimmed through this, and it seems pretty good! (seems like you are able to pull the code samples from the notebooks?)

File: llama-index-integrations/llms/llama-index-llms-anthropic/pyproject.toml (hunk 1 above)
Comment (anoopshrma) [reply llama-index-integrations/llms/llama-index-llms-anthropic/pyproject.toml:30]: I think yeah for llamahub we don't need to update the version but for PYPI to reflect the new readme update we do.

**Comment by anoopshrma** [comment]
Yes used the notebook code samples to generate the readme 😄

Initially I was trying to generate the readme using the `base.py`   but results were not that good in comparison to when I tried with notebooks. So I stick with that 🚀

**Review by logan-markewich — APPROVED** [review]

**Pull request merged**